cruise/CruiseInformation.csv

```diff
@@ -42,13 +42,15 @@ AT27,Atlantis,2015-04-28T16:00:00,2015-05-14T12:26:00,Pioneer 4
 TN-323,Thomas G. Thompson,2015-05-29T16:00:00,2015-06-13T04:00:00,Papa 3 #TO BE DELETED
 TN323,Thomas G. Thompson,2015-05-29T16:00:00,2015-06-13T04:00:00,Papa 3
 PS1506A,Pacific Storm & Elakha ,2015-06-03T00:00:00,2015-06-03T23:59:59,Endurance 3b
+CT20150612,Connecticut,2015-06-12T00:00:00,2015-06-12T23:59:59,Pioneer ISSP recovery
```

Comment:
looks good - verified as much as possible via the link that Collin 'slacked' to data team.

Reply:
Looks good.